kolibri/core/assets/src/views/buttons-and-links/k-button.vue

```diff
@@ -6,9 +6,7 @@
     :type="type"
     :disabled="disabled"
     @click="handleClick"
-  >
-    {{ text }}
-  </button>
+  ><!-- no whitespace -->{{ text }}<!-- no whitespace --></button>
```

Comment:
Curious what was the reasoning behind this, since [this rule](https://github.com/learningequality/kolibri/blob/release-v0.8.x/frontend_build/src/webpack.config.base.js#L101) should be taking care of that. And I cannot see the whitespace in the buttons when I inspect the element in FF.

Reply:
I'm not sure, but I ran into a bug where there were underlined spaces around a link...